plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -594,7 +594,7 @@ func podName(pod *v1.Pod) string {
 // PodFitsResources checks if a node has sufficient resources, such as cpu, memory, gpu, opaque int resources etc to run a pod.
 // First return value indicates whether a node has sufficient resources to run a pod while the second return value indicates the
 // predicate failure reasons if the node has insufficient resources to run the pod.
-func PodFitsResources(pod *v1.Pod, meta algorithm.PredicateMetadata, nodeInfo *schedulercache.NodeInfo) (bool, []algorithm.PredicateFailureReason, error) {
+var PodFitsResources algorithm.FitPredicateFunction = func(pod *v1.Pod, meta algorithm.PredicateMetadata, nodeInfo *schedulercache.NodeInfo) (bool, []algorithm.PredicateFailureReason, error) {
```

Comment:
I don't like this change. Don't change like this.

Reply:
I can keep it as before, but every time you use it you have to use it like this: `algorithm.FitPredicateFunction(PodFitsResources)`.

Reply:
+1 to what @dixudx wrote, I don't like changing functions to vars.